rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/action/TbSaveToCustomCassandraTableNode.java

```diff
@@ -63,8 +63,7 @@
         nodeDetails = "Administrator should set the custom table name without prefix: <b>cs_tb_</b>. <br>" +
                 "Administrator can configure the mapping between the Message field names and Table columns name.<br>" +
                 "<b>Note:</b>If the mapping key is <b>$entity_id</b>, that is identified by the Message Originator, then to the appropriate column name(mapping value) will be write the message originator id.<br><br>" +
-                "If specified message field does not exist or is not a JSON Primitive, the outbound message will be routed via <b>failure</b> chain," +
-                " otherwise, the message will be routed via <b>success</b> chain.",
+                "Output connections: <code>Failure</code> - if specified message field does not exist or is not a JSON Primitive, otherwise <code>Success</code>.",
```

Comment:
```suggestion
                "Output connections: <code>Failure</code> - if specified message field does not exist or it is not a JSON Primitive, otherwise <code>Success</code>.",
```